resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/BasicDriverFeatureStep.scala

```diff
@@ -99,6 +99,9 @@ private[spark] class BasicDriverFeatureStep(conf: KubernetesDriverConf)
       conf.sparkConf.getInt(BLOCK_MANAGER_PORT.key, DEFAULT_BLOCKMANAGER_PORT)
     )
     val driverUIPort = SparkUI.getUIPort(conf.sparkConf)
+    // To avoid a new compile dependency, we use the string config name and default value.
+    val driverSparkConnectServerPort =
+      conf.sparkConf.getInt("spark.connect.grpc.binding.port", 15002)
```

Comment:
I think you can use `DEFAULT_SPARK_CONNECT_SERVER_PORT` here.

Reply:
And maybe extracting `spark.connect.grpc.binding.port` out to `Constants` would make sense too.